library/alloc/src/alloc.rs

```diff
@@ -31,8 +31,14 @@ unsafe extern "Rust" {
     #[rustc_std_internal_symbol]
     fn __rust_alloc_zeroed(size: usize, align: usize) -> *mut u8;
 
+    #[rustc_nounwind]
+    #[rustc_std_internal_symbol]
+    #[cfg(not(bootstrap))]
+    fn __rust_no_alloc_shim_is_unstable_v2();
     #[rustc_std_internal_symbol]
+    #[cfg(bootstrap)]
     static __rust_no_alloc_shim_is_unstable: u8;
```

Comment:
`#[cfg(bootstrap)]` is no longer necessary: https://blog.rust-lang.org/inside-rust/2025/05/29/redesigning-the-initial-bootstrap-sequence/